pages/guide.py

```diff
@@ -98,15 +78,14 @@ def process_post_data(self, **kwargs):
         breaking_change=self.form.get('breaking_change') == 'on',
         blink_components=blink_components,
         tag_review_status=processes.initial_tag_review_status(feature_type))
-    feature_entry.put()
+    key: ndb.Key = feature_entry.put()
```

Comment:
Why introduce a local variable for this result if it is not used?

Reply:
This was how the key was referenced when creating the old Feature entity to reference in other places. This variable is used a bit further down, but it is not used in all the places it could be. I've refactored the code to use this variable anywhere it should be referenced.